iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -345,6 +342,23 @@ private async Task ReceiveMessageAndCompleteAsync()
             }
         }
 
+        private (Message, string) PrepareMessage(int messageId)
```

Comment:
This will make disposing `Message` difficult because of the tuple. Thoughts?

Reply:
Actually we don't seem to be using the payload in our sample. I'll update this.
We were disposing the message once the "send" operation was complete, after this change we can use `using`